src/components/deviceMap/deviceMap.js

```diff
@@ -186,13 +186,13 @@ class DeviceMap extends Component {
   render() {
     const { BingMapKey } = this.props;
     return (
-      <DashboardPanel title={lang.DEVICELOCATION} className="map-container">
+      <DashboardPanel title={lang.DEVICELOCATION} indicator={this.props.showSpinner} className="map-container">
         <Row>
           <RegionDetails {...this.props} />
           <Col md={9} className="bing-map">
             {
-              BingMapKey === config.STATUS_CODES.STATIC 
-                ? <div className="static-bing-map"/> 
+              BingMapKey === config.STATUS_CODES.STATIC
+                ? <div className="static-bing-map"/>
```

Comment:
Is the static-map bug also fixed?

Reply:
No. This PR focused on the spinner for mapControl and interval.